### ISSUE TITLE
Signup accessibility issues

### ISSUE THREAD

**Issue by PrivacyDragon** [tutao/tutanota#3623]
- [X] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [X] I've searched and did not find a similar issue.

**Bug in web app**
When using a screenreader the signup form is very unclear. It can read the form correctly kinda, but it continues to read the rest of the page, which is the login form. So, that is not really nice. Furthermore, I think it is not really good that the buttons are labeled 'select' without their plan. So, a screenreader user cannot be completely sure what plan they select. I think. (Though I am myself not a screenreader user.). 
Overall, it is usable, but it could probably use some improvements.

**To Reproduce**
1. Turn on a screenreader. I used Orca on Linux.
2. Go to the signup page, preferably with your eyes closed.
3. Hear what the screenreader gives as output.
4. It is not really nice.

**Expected behavior**
Screenreader users should not have any troubles with signing up...

**Desktop (please complete the following information):**
 - OS: OpenSuse Leap 15.3
 - Browser: Brave 1.22.71 Chromium: 89.0.4389.114 (Official Build) unknown (64-bit)

**Fix**
Have a page for only the signing up form and give better labels to the 'select' buttons.

**Test Notes**
- [ ] Turn on talk Back in android and navigate to signup page.
- [ ] Text fields and all other information in signup page are reading only one time(instead of twice)

**Comment by charlag** [comment]
@PrivacyDragon thanks a lot for reporting! We will re-test signup page and try to fix those issues
